### DISCUSSION TITLE
Doubt regarding the project development

### DISCUSSION THREAD

**Discussion by reesshhoo** [github/india#281] [Students]
I'm not very well versed with technology to make working prototype applications which is necessary for this internship, since I'm from Accounting background . Will it be possible for me to make a project after completing the self-learning sessions?

**Comment by RomaJadhav** [comment]
> I'm not very well versed with technology to make working prototype applications which is necessary for this internship, since I'm from Accounting background . Will it be possible for me to make a project after completing the self-learning sessions?

Hi i was tensed regarding the same but once you start self learning one of the modules had a step by step exercise on how to create a wordpress website with azure so i think the next modules would have the same kind of training.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
